### PR TITLE
Feature/DTGB-406: First implementation of the standard footer.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/11-base/fonts/_fonts.scss [hunk 1]
```diff
@@ -2,5 +2,5 @@
 /// This file defines fonts.
 ////
 
-@import url('https://fonts.googleapis.com/css?family=Fira+Sans:400,600,700'); // sass-lint:disable-line no-url-protocols
+@import url('https://fonts.googleapis.com/css?family=Fira+Sans:400,600,700,800'); // sass-lint:disable-line no-url-protocols
```
Comment (delrueba) [inline styleguide/components/11-base/fonts/_fonts.scss]: I know this is not your decision, but has design been made aware that we are adding an extra font weight?
At this rate, we'll end up with all font variants before the end of this month. Despite a clear resolution not to.

**Review by delrueba — APPROVED** [review]

File: styleguide/components/31-molecules/blocks/disclaimer-block/disclaimer-block.twig [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
-      "<a href='https://stad.gent/werken-ondernemen/werken/werken-bij-de-stad-gent/vacatures-en-solliciteren'>Vacancies</a>",
-      "<a href='https://stad.gent/over-gent-en-het-stadsbestuur/over-gent/kaarten-cijfers-en-data/gent-kaart/stadsplan;>City plan</a>",
-      "<a href='http://persruimte.stad.gent/'>Press room City of Ghent</a>",
-      "<a href='https://stad.gent/over-gent-en-het-stadsbestuur/vragen-suggesties-en-meldingen/juridische-info-en-privacy'>Legal info and privacy</a>"
-    ]
-  }
-%}
+<div class="dg-block-disclaimer">
+  {% include 'h3'with {
+    'h3_text': 'Nuttige links'
```
Comment (Jeroen005) [inline styleguide/components/31-molecules/blocks/disclaimer-block/disclaimer-block.twig]: Always use English copy in stijlgids.

File: styleguide/components/31-molecules/blocks/post-address-block/post-address-block.twig [hunk 3]
```diff
@@ -1,6 +1,6 @@
-<div class="dg-block-post-address island">
+<div class="dg-block-post-address">
   {% include 'h3'with {
-    'h3_text': 'Mailing address'
+    'h3_text': 'Postadres'
```
Comment (Jeroen005) [inline styleguide/components/31-molecules/blocks/post-address-block/post-address-block.twig]: English

File: styleguide/components/31-molecules/islands/gentinfo-island/gentinfo-island.twig [hunk 4]
```diff
@@ -1,13 +1,28 @@
...
   } %}
+  <div class="gentinfo-block-content">
+    {% include '@h2'
+      with {
+        'h2_text': 'Gentinfo'
+      }
+    %}
+    {% include '@paragraph'
+      with {
+        'text': 'Gentinfo is bereikbaar van maandag t.e.m. zaterdag van 08:00 tot 19:00'
```
Comment (Jeroen005) [inline styleguide/components/31-molecules/islands/gentinfo-island/gentinfo-island.twig]: English

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/31-molecules/blocks/disclaimer-block/disclaimer-block.twig (hunk 2 above)
Comment (meirege) [reply styleguide/components/31-molecules/blocks/disclaimer-block/disclaimer-block.twig]: Changed it to English.

File: styleguide/components/31-molecules/blocks/post-address-block/post-address-block.twig (hunk 3 above)
Comment (meirege) [reply styleguide/components/31-molecules/blocks/post-address-block/post-address-block.twig]: Changed it to English.

File: styleguide/components/31-molecules/islands/gentinfo-island/gentinfo-island.twig (hunk 4 above)
Comment (meirege) [reply styleguide/components/31-molecules/islands/gentinfo-island/gentinfo-island.twig]: Changed it to English.

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**